tron/bin/check_tron_jobs.py

```diff
@@ -96,19 +95,23 @@ def compute_check_result_for_job_runs(client, job, job_content):
         url_index,
         relevant_job_run['id'],
     )
-    action_runs = client.job(job_run_id.url, include_action_runs=True)
+
     # A job action is like MASTER.foo.1.step1
     actions_expected_runtime = job_content.get('actions_expected_runtime', {})
     relevant_action = get_relevant_action(
-        action_runs=action_runs["runs"],
+        action_runs=relevant_job_run["runs"],
         last_state=last_state,
         actions_expected_runtime=actions_expected_runtime
     )
     action_run_id = get_object_type_from_identifier(
         url_index,
         relevant_action['id'],
     )
-    action_run_details = client.action_runs(action_run_id.url, num_lines=10)
+
+    if last_state in (State.STUCK, State.FAILED):
```

Comment:
is this the right set of states in which to fetch the action runs?

Reply:
You should also include UNKNOWN.

Reply:
Done